STYLE-GUIDE.md

```diff
@@ -0,0 +1,22 @@
+# Exercism's Style Guide
+
+This document acts as a Style Guide for the language and wording used in exercises.
+
+## General principle
+
+All content should be written in US english. 
+In the future other translations may occur, but the "official" Exercism natural langauge is US English.
+
+## Explain or substitude mathmatical and esotoric terms.
+
+In any place that mathematical terms are used they should be explained or substituted out for terms that require less domain knowledge.
+
+Examples:
+- Rather than using "natural numbers", we should use "positive integers" or "zero and positive integers". 
+- If we want to use the phrase "rational numbers", it must be explained in the introduction to the exercise.
+- Rather than using the word range (which can have different meanings in different contexts) use "greater than x and less than y".
```

Comment:
Depending on the context `x < _ < y` may be better (as in more readable), imo.

Reply:
I prefer both.

```
x < ? < y (greater than x and less than y)
```

Reply:
```suggestion
- Rather than using the word range (which can have different meanings in different contexts) use "x < ? < y (greater than x and less than y)".
```